smother/control.py

```diff
@@ -81,6 +86,13 @@ def write(self, file_or_path, append=False, timeout=10):
         multiprocess test environment.
         """
         if isinstance(file_or_path, six.string_types):
+            if self.coverage and self.coverage.config.parallel:
```

Comment:
I was imitating how coverage does it. They wait until the last possible moment (which is during write) to determine the output file name. It made sense to do it that way in the coverage source so that it wouldn't interfere elsewhere, but smother isn't as complicated, so it is possible to determine the filename later. I like it this way still, though, because write is called from a lot of places and it makes sense to me that the output filename should be automatically modified if the parallel option is being used.